### PR TITLE
Update Dropdown.tsx

### PR DESCRIPTION
**Pull request by zhukovka**
recompute selected option if selected option is changed from next props

### CODE REVIEW

File: src/components/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -89,15 +89,17 @@ export class Dropdown extends React.Component<DropdownProps, State> {
   }
 
   static getDerivedStateFromProps(nextProps, state) {
+    if (state.selectedOption && nextProps.initialSelectedId !== state.selectedOption.id || !state.selectedOption) {
```
Comment (jonathanadler) [inline src/components/Dropdown/Dropdown.tsx:92]:
```suggestion
    if (!state.selectedOption || nextProps.initialSelectedId !== state.selectedOption.id) {
```

File: src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -89,15 +89,17 @@ export class Dropdown extends React.Component<DropdownProps, State> {
...
+      return {
+        selectedOption:
+          nextProps.options.find(
+            option => option.id === nextProps.initialSelectedId,
+          ) || null,
+      };
+    }
     if (state.selectedOption) {
       return null;
     }
```
Comment (jonathanadler) [inline src/components/Dropdown/Dropdown.tsx:102]: In this case, we should always return null, and not only if `state.selectedOption`

**Comment by shlomitc** [comment]
is this PR still relevant?

**Comment by jonathanadler** [comment]
Closing for now

**Pull request closed without merging**